cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonConstants.java

```diff
@@ -71,4 +71,32 @@ private CommonConstants() {
     public static final String INTERNAL_CONCATENATOR = "=";
     public static final String CONCATENATOR = "x0000";
     
+    // Header for API uses
+    public static final String INSTANCE_HEADER = "#####\n" 
```

Comment:
Of course, it's a great idea in order to use it in the future :)
Fixed in 8f82548